### PR TITLE
Add some fixes for the FunctionalTests and fix typing on a UIAccessibilityTextFieldElement

### PR DESCRIPTION
**Pull request by tirodkar**

### CODE REVIEW

**Comment by khandpur** [comment]
Noticing lots of failures with this PR: https://travis-ci.org/google/EarlGrey/builds/205961731
Please investigate.

**Comment by tirodkar** [comment]
Looks better now.

**Comment by khandpur** [comment]
Not really. The gesture tests are consistently failing.

**Comment by tirodkar** [comment]
Good to go now.

File: Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m [hunk 1]
```diff
@@ -464,6 +464,20 @@ - (void)testTypingOnTextFieldInUIInputAccessory {
...
+      performAction:grey_tap()];
+  Class accessibilityTextFieldElemClass = NSClassFromString(@"UIAccessibilityTextFieldElement");
+  [[EarlGrey selectElementWithMatcher:grey_allOf(grey_accessibilityValue(@"Text Field"),
+                                                 grey_kindOfClass(accessibilityTextFieldElemClass),
+                                                 nil)]
+      performAction:grey_clearText()];
+  [[EarlGrey selectElementWithMatcher:grey_allOf(grey_accessibilityValue(@"Text Field"),
+                                                 grey_kindOfClass(accessibilityTextFieldElemClass),
+                                                 nil)]
+      performAction:grey_replaceText(@"")];
```
Comment (sid-github) [inline Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m]: we are not actually typing anything? just clear and replace? Also missing assertions?

Comment (tirodkar) [inline Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m]: Yeah, this only checks if clearText and replaceText won't throw errors if we try to match with an element of `UIAccessibilityTextFieldElement`.

Comment (sid-github) [inline Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m]: In that case I suggest renaming the test to something like `testClearAndReplaceWorksWithAccessibilityElements`

File: Tests/FunctionalTests/TestRig/Sources/FTRNetworkTestViewController.m [hunk 2]
```diff
@@ -70,16 +69,11 @@ - (void)verifyReceivedData:(NSData *)data {
   }
 }
 
+
```
Comment (khandpur) [inline Tests/FunctionalTests/TestRig/Sources/FTRNetworkTestViewController.m]: excess new line.

File: Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m (hunk 1 above)
Comment (tirodkar) [inline Tests/FunctionalTests/Sources/FTRKeyboardKeysTest.m]: Done.

File: Tests/FunctionalTests/TestRig/Sources/FTRNetworkTestViewController.m (hunk 2 above)
Comment (tirodkar) [inline Tests/FunctionalTests/TestRig/Sources/FTRNetworkTestViewController.m]: Done.

File: EarlGrey/Action/GREYActions.m [hunk 3]
```diff
@@ -332,14 +347,19 @@ + (void)grey_webSetText:(id)element text:(NSString *)text {
...
                  nil);
   NSString *replaceActionName = [NSString stringWithFormat:@"Replace with text: \"%@\"", text];
   return [GREYActionBlock actionWithName:replaceActionName
                              constraints:constraints
                             performBlock:^BOOL (id element, __strong NSError **errorOrNil) {
     if ([element grey_isWebAccessibilityElement]) {
       [GREYActions grey_webSetText:element text:text];
+    } else if ([element isKindOfClass:accessibilityTextFieldElementClass]) {
+      UIAccessibilityTextFieldElement *textFieldElement =
+      (UIAccessibilityTextFieldElement *)element;
```
Comment (sid-github) [inline EarlGrey/Action/GREYActions.m]: +4 indent

Comment (tirodkar) [inline EarlGrey/Action/GREYActions.m]: Done! Thanks.

**Comment by tirodkar** [comment]
@sid-github This is good to go. Cancelled the last CI run since the test was only a formatting one.

**Pull request merged**